### PR TITLE
Fix hyperlink to tf.Tensor guide.  

### PR DESCRIPTION
**Pull request by jraman**
cc: @jvishnuvardhan 

Followup to #36775.
 
 * Fix hyperlink to tf.Tensor guide. 
 * Add brief intro of eager and graph modes.

### CODE REVIEW

**Comment by jraman** [comment]
@googlebot I signed it!

**Comment by jraman** [comment]
Bump.  Could you please review?  Thanks.

File: tensorflow/python/framework/ops.py [hunk 1]
```diff
@@ -291,22 +291,23 @@ def disable_tensor_equality():
...
 
-  Each element in the Tensor has the same data type, and the data type is always
-  known.
+  TensorFlow supports two execution modes: eager and graph.  In eager
+  mode, operations are evaluated immediately.  In graph mode, a
+  computational graph is constructed for later evaluation.
 
-  In eager execution, which is the default mode in TensorFlow, results are
-  calculated immediately.
+  Eager mode is the default mode in TensorFlow.  In the example below,
```
Comment (lamberta) [inline tensorflow/python/framework/ops.py]:
```suggestion
  TensorFlow defaults to eager execution.  In the example below,
```

File: tensorflow/python/framework/ops.py [hunk 2]
```diff
@@ -291,22 +291,23 @@ def disable_tensor_equality():
...
 
-  * a data type (float32, int32, or string, for example)
+  * a single data type (float32, int32, or string, for example)
   * a shape
 
-  Each element in the Tensor has the same data type, and the data type is always
-  known.
+  TensorFlow supports two execution modes: eager and graph.  In eager
+  mode, operations are evaluated immediately.  In graph mode, a
+  computational graph is constructed for later evaluation.
```
Comment (lamberta) [inline tensorflow/python/framework/ops.py]:
```suggestion
  TensorFlow supports eager execution and graph execution.  In eager
  execution, operations are evaluated immediately.  In graph execution, a
  computational graph is constructed for later evaluation.
```

**Review by lamberta — CHANGES_REQUESTED** [review]
Thanks. Looks good, but avoid "mode" when describing eager execution and graph execution.

**Comment by gbaned** [comment]
@jraman Can you please check lamberta's comments and keep us posted? Thanks!

**Comment by jraman** [comment]
@lamberta, thank you for your review.  I have addressed your comments.

cc: @gbaned

**Review by lamberta — APPROVED** [review]
Thanks!

**Comment by jraman** [comment]
Bump.  Could you merge please?  Thanks.
cc: @gbaned

**Comment by lamberta** [comment]
Just checked on it. It's in-flight but this process can take while because of our internal CI (cl/299268631)

**Comment by jraman** [comment]
@lamberta, @gbaned 
Any update?  Sorry to bother you, but it's almost four days since CI was last checked.  Thanks.

**Pull request merged**